apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCard.tsx

```diff
@@ -21,10 +22,30 @@ export function JourneyCard({ journey }: JourneyCardProps): ReactElement {
     year: isThisYear(date) ? undefined : 'numeric'
   })
 
+  const breakpoints = useBreakpoints()
+  const border = breakpoints.sm ? 12 : 0
+
   return (
     <Card
+      aria-label="journey-card"
       variant="outlined"
-      sx={{ borderRadius: 0, px: 6, py: 4, border: 'none' }}
+      sx={{
+        px: 6,
+        py: 4,
+        borderRadius: 0,
+        borderColor: 'divider',
+        borderBottom: 'none',
+        '&:first-child': {
+          borderTopLeftRadius: border,
```

Comment:
I'm not a fan of using breakpoints for styling. We should reserve useBreakpoints where codeflow is affected by responsiveness. You should use the expressive syntax inline that mui provides for this.

```suggestion
          borderTopLeftRadius: { xs: 0, sm: 12 },
```